receiver/statsdreceiver/README.md

```diff
@@ -38,7 +38,7 @@ The Following settings are optional:
 - `timer_histogram_mapping:`(default value is below): Specify what OTLP type to convert received timing/histogram data to.
```

Comment:
Unfortunately this field name is now misleading since it's a mapping for `timer`s, `histogram`s, and `distribution`s.  At least, the description should be updated

```
Specify what OTLP type to convert received timing/histogram/distribution data to.
```